tests/integration/server/courseInstance/courseInstance.service.test.ts

```diff
@@ -199,6 +201,53 @@ describe('Course Instance Service', function () {
         });
       });
     });
+    describe('Ordering', function () {
+      it('should order by area => prefix => integer => letter', function () {
+        for (let i = 0; i < result.length; i++) {
```

Comment:
Would a `forEach` be a better fit here, instead of `for` loop?

Reply:
I usually prefer to use the `for` loop for cases where we need to compare adjacent values (`result[i]` and `result[i +1]`) since it makes it a little more explicit where each of those values is coming from. I don't think it really makes a difference either way, but did you have a reason in mind for using `forEach`?

Reply:
My motivation for suggesting `foreach` was more semantics (i.e: foreach <thing> in <things> rather than indexing `things[i]`). `Array.prototype.foreach` [does provide a second argument to the callback](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach#parameters) for the index so you could index adjacent values like that if you wanted to. But I don't feel particularly strongly about it, it was just a passing suggestion.